net/keepalived/files/etc/init.d/keepalived

```diff
@@ -256,6 +256,21 @@ print_track_bfd_indent() {
 	printf '\n' >> "$KEEPALIVED_CONF"
 }
 
+print_unicast_peer_indent() {
```

Comment:
Can you please move this change into a separate commit?
I assume that does have nothing to do with sync

Reply:
This one is actually needed.

Whenever keepalived peers are configured, to sync files, we also configure peers sync configuration. Instead of IP only peers can be saved with the name `master` or `backup` or 'other name`. which would be used as a reference in the vrrp_instance configuration same as track_interface, track_scripts, etc. And the same peer config is used in rsync.sh and inotify also.

In the `sender` (master) node, the peer saves SSH configuration, and additional files to sync.
In the `receiver` (backup) node, the peer saves SSH public key configuration.